keras_nlp/models/whisper/whisper_tokenizer.py

```diff
@@ -113,6 +113,8 @@ def __init__(
         self.translate_token_id = special_tokens[translate_token]
         self.transcribe_token_id = special_tokens[transcribe_token]
 
+        self.end_token_id = self.eos_token_id
```

Comment:
This is kind of an awkward line. Can we just update the whole layer to use `end_token_id` and `start_token_id`? That will be more consistent with the rest of the library.